cli/cmd/service/list.go

```diff
@@ -16,18 +17,33 @@ func ListCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
+			if project != "" {
+				// If project is specified, list services for that project
+				res, err := client.ListProjectServices(cmd.Context(), &adminv1.ListProjectServicesRequest{
+					OrganizationName: ch.Org,
+					ProjectName:      project,
+				})
+				if err != nil {
+					return err
+				}
+
+				ch.PrintProjectMemberServices(res.Services)
+				return nil
+			}
+
 			res, err := client.ListServices(cmd.Context(), &adminv1.ListServicesRequest{
```

Comment:
So with the changes inside `ListServices` this now only lists services with an org role, right? I think people might expect `rill service list` to list all services in the org, both those with an org role and those that are project-level. 

Could it maybe list all services and have a) the org role column say "none" if there isn't one, b) maybe have a bool column for "has project roles"?